rasa/test.py

```diff
@@ -154,7 +155,7 @@ def test_core(
     rasa.utils.common.run_in_loop(test(stories, _agent, out_directory=output, **kwargs))
 
 
-def test_nlu(
+async def test_nlu(
```

Comment:
this function is also called from [here](https://github.com/RasaHQ/rasa/blob/bc76915a8fbd40f505f75c8246f03f26caaa0323/rasa/test.py#L104). Seems we need another `rasa.utils.common.run_in_loop` here